api/src/main/java/com/cloud/agent/api/to/VirtualMachineTO.java

```diff
@@ -65,6 +65,7 @@
     String uuid;
     String bootType;
     String bootMode;
+    boolean enterHardwareSetup;
```

Comment:
@DaanHoogland Is hardware setup same as boot setup?

Reply:
in hindsite this name should have been something like "bootIntoFirmware". many names have passed the catwalk. on resource level it is called enterHardwareSetup on API it is bootIntoSetup.